astrodata/test/test_descriptor_values.py

```diff
@@ -0,0 +1,60 @@
+
+import os
+import pytest
+import glob
+import warnings
+import datetime
+
+import astrodata
+import gemini_instruments
+from .conftest import test_path
+
+
+try:
+    path = os.environ['TEST_PATH']
+except KeyError:
+    warnings.warn("Could not find environment variable: $TEST_PATH")
+    path = ''
+
+if not os.path.exists(path):
+    warnings.warn("Could not find path stored in $TEST_PATH: {}".format(path))
+    path = ''
+
+archivefiles = glob.glob(os.path.join(path, "Archive/", "*fits"))
+
+# Cleans up a fake file created in the tests in case it's still there
+cleanup = os.path.join(path, 'created_fits_file.fits')
+if os.path.exists(cleanup):
+    os.remove(cleanup)
+
+
+# Fixtures for module and class
+@pytest.fixture(scope='class')
```

Comment:
This fixture does not seem to be doing nothing. If it is the case, I would remove it.